### PR TITLE
TE5-T269 Swift Basic

### PR DESCRIPTION
**Pull request by buddhdev-jay**

### CODE REVIEW

File: Swift_Basic.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+//Closure
+let simpleClosure:(String) -> (String) = { name in
+    let greeting = "Hello Jay " + "Welcome to Swift"
+    return greeting
+}
+let result1 = simpleClosure("Hello JavaTpoint")
+print(result1)
+
+
```
Comment (purvesh-d) [inline Swift_Basic.playground/Contents.swift:302]: Remove extra empty lines

**Review by purvesh-d — CHANGES_REQUESTED** [review]
Please resolve comment.

File: Swift_Basic.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+        print("The largest number is ",  n2)
+    }
+    else {
+        print("The largest number is ", n3)
+    }
+}
+//guard
+guard true else {
+    print("Condition is not satisfied.")
+
```
Comment (purvesh-d) [inline Swift_Basic.playground/Contents.swift:181]: Remove extra empty lines from here as well from all the files which are included in this PR.

File: Swift_Basic.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,621 @@
...
+    print(num1)
+    num1 + 1
+}
+//Repeat While
+num1 = 5
+repeat
+{
+    print(num1)
+    num1 + 1
+}while num1<5
```
Comment (purvesh-d) [inline Swift_Basic.playground/Contents.swift:222]: Code formating required. Try select all code and then Ctrl+ i to formate it.

File: Swift_Exercise.playground/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+{
+    string4=string3
+}
+else
+{
+    let index1=string3.index(string3.startIndex, offsetBy: 2)
+    string4 = String(string3[..<index1])
+}
+print(string4)
+//2.Write a Swift program to create a new string without the first and last character of a given string
```
Comment (purvesh-d) [inline Swift_Exercise.playground/Contents.swift:86]: Why this commented code added ?

**Review by purvesh-d — CHANGES_REQUESTED** [review]
- Resovle all comments
- There should be single commit in single PR.

File: Swift_Basic.playground/Contents.swift (hunk 1 above)
Comment (buddhdev-jay) [reply Swift_Basic.playground/Contents.swift:321]: okk

File: Swift_Exercise.playground/Contents.swift [hunk 5]
```diff
@@ -0,0 +1,578 @@
...
+    }
+}
+//2.Write a Swift program to create a new array with double the lenght of a given array of integers and its last element is the same as the given array. The given array will be length 1 or more. By default, a new integer array contains all 0's.
+var array2 = Array(repeating:0 , count:array1.count * 2)
+print (array2[0])
+array2[array2.count-1] = array1[array1.count-1]
+print(array2)
+//3.Write a Swift program to create a new array, taking first two elements from a given array of integers. If the length of the given array is less than 2 use the single element of the given array
+var two_number_array = [Int]()
+if array1.count > 2{
```
Comment (purvesh-d) [inline Swift_Exercise.playground/Contents.swift:134]: Code formatting

**Review by purvesh-d — CHANGES_REQUESTED** [review]

**Pull request merged**